definitions/horizontal_scaling.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Horizontal Scaling
+status: Feedback Appreciated
+category: Concept
+---
+
+## Horizontal Scaling
+
+### What it is
+
+In [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) perspective Horizontal Scaling is type of [scalability](https://github.com/cncf/glossary/blob/main/definitions/scalability.md) whose approach is to add additional instances instead of moving into a larger instance size.
+
+For example, let us consider we have a system of 4GB RAM and we want to increase it's capacity to 16GB RAM, Horizontal Scaling means doing this by making a system of 4 X 4GB RAM rather than switching to a 16GB RAM system.
+
+It is an approach to enhance the performance of the server [node](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) by adding new instances of the server to already existing servers in order to distribute the workload equally. In simple words it aims to decrease the server's load rather than expanding capacity of the individual server.
+
+Load balancing, clustering, and distributed file systems are some of the crucial and popular strategies of horizontal scaling. It caters to portioning of data where each [node]((https://github.com/cncf/glossary/blob/main/definitions/nodes.md)) contains single part of the data.
```

Comment:
I think it would be best to remove this line. It gets too into the details.

Reply:
@JasonMorgan Yeah sure I also had this thought, as it aims to give a basic idea to the person reading. I felt this was a bit too much of info for someone starting. Apart from that the terms may also be difficult for a newbie to understand.
I was unsure myself while adding it.